readthedocs/projects/models.py

```diff
@@ -1483,6 +1483,8 @@ class EnvironmentVariable(TimeStampedModel, models.Model):
         help_text=_('Project where this variable will be used'),
     )
 
+    objects = RelatedProjectQuerySet.as_manager()
```

Comment:
Did we not need this before? We only appear to be calling `all` on it.

Reply:
Probably we needed it, but nothing was using it to list these related objects.